### PR TITLE
Hwo/lesson1

### PR DESCRIPTION
**Pull request by Felburg**
Took a tutorial at GitHub.

### CODE REVIEW

**Comment by Felburg** [comment]
Waiting for approval.

Another training underway.

**Comment by lukashlobil** [comment]
Please move the files into lesson1 folder along with the original lesson1.js file. The file doesn't have proper .js file extension. Please also remove the file stating your progress in git

Felburgs comment:
- file moved to lesson1 folder
- file renamed #1 Homework → Homework_001.js
- file stating Git training progress removed

File: #1 Homework
Comment (lukashlobil) [inline #1 Homework:26]: correct code, incorrect comment

File: #1 Homework
Comment (lukashlobil) [inline #1 Homework:10]: Please use comments in english, it will help you to memorise correct terms faster

**Comment by lukashlobil** [comment]
Congratulations, it's correct, I have just a couple of suggestions how to make the code more readable, please see comments and submit new PR

File: #1 Homework
Comment (lukashlobil) [inline #1 Homework:10]: If you use a certain value, that you need to access via a getter, it is better to use it as a variable

**Review by lukashlobil — CHANGES_REQUESTED** [review]

File: #1 Homework
Comment (lukashlobil) [reply #1 Homework:10]: so here it would be better to have a variable called ie. current = array[i] and then replace all of the array[i] with current

File: #1 Homework
Comment (lukashlobil) [inline #1 Homework:20]: remove

File: #1 Homework
Comment (lukashlobil) [inline #1 Homework:18]: array[d] < array[i]

**Comment by Felburg** [comment]
Cancelling Pull request as it needs more love.

File: #1 Homework
Comment (Felburg) [reply #1 Homework:26]: - comments translated into English
- comments specified to show exact outcoms

File: #1 Homework
Comment (Felburg) [reply #1 Homework:10]: - all comments translated

Comment (Felburg) [reply #1 Homework:10]: Updated:
- var j = array[i];

File: #1 Homework
Comment (Felburg) [reply #1 Homework:20]: Please review and if issue remains please specify.

Comment (Felburg) [reply #1 Homework:20]: Updated as by the further comment.
- var maxd removed as not necessary

File: #1 Homework
Comment (Felburg) [reply #1 Homework:18]: Code updated as oer comment.

**Comment by Felburg** [comment]
Code adjusted, working on creating new branch and pull req.

Thank you for relevant comments.

**Pull request closed without merging**
